cmd/template/cluster/flag.go

```diff
@@ -29,12 +29,14 @@ const (
 	flagTemplateDefaultNodepool = "template-default-nodepool"
 
 	// nodepool flags
-	flagAvailabilityZones    = "availability-zones"
-	flagAWSInstanceType      = "aws-instance-type"
-	flagNodepoolName         = "nodepool-name"
-	flagNodesMax             = "nodex-max"
-	flagNodesMin             = "nodex-min"
-	flagNumAvailabilityZones = "num-availability-zones"
+	flagAvailabilityZones                   = "availability-zones"
+	flagAWSInstanceType                     = "aws-instance-type"
+	flagNodepoolName                        = "nodepool-name"
+	flagNodesMax                            = "nodex-max"
+	flagNodesMin                            = "nodex-min"
+	flagNumAvailabilityZones                = "num-availability-zones"
+	flagOnDemandBaseCapacity                = "on-demand-base-capacity"
+	flagOnDemandPercentageAboveBaseCapacity = "on-demand-precentage-above-base-capacity"
```

Comment:
I'm not sure if those flags are descriptive enough. I'd like to have something the CR structure better. But my practical approach may lead to really ugly UX. Summoning @giantswarm/sig-ux here as they may have some good ideas.

Reply:
it is probably time to move to config files from flags

Reply:
Do you want to wait for sig ux for this one, or create a main issue where we can tackle this properly, as Adidas is wainting for me to release the other change too

Reply:
I don't want to make the decision here as I'm not involved enough in development of the plugin. I'm just alerted.